src/Nethermind/Nethermind.JsonRpc.Test/Sockets/RandomString.cs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+
+namespace Nethermind.JsonRpc.Test.Sockets;
+
+public class RandomString
```

Comment:
Wouldn't just some method suffice?

Reply:
Initially all "RandomX" classes were methods, but I moved them to classes for clarity. I'll undo the change.